### ISSUE TITLE
Add support for transitive dependency inclusion during generateKar task

### ISSUE THREAD

**Issue by eschizoid** [lburgazzoli/gradle-karaf-plugin#7]
`maven-karaf-pluggin` supports including transitive dependencies during the creation of the kar file:

````

		<pluginManagement>
			<plugins>
				<plugin>
					<groupId>org.apache.karaf.tooling</groupId>
					<artifactId>karaf-maven-plugin</artifactId>
					<version>${karaf.version}</version>
					<extensions>true</extensions>
				</plugin>
			</plugins>
		</pluginManagement>
		<plugins>
			<plugin>
				<groupId>org.apache.karaf.tooling</groupId>
				<artifactId>karaf-maven-plugin</artifactId>
				<configuration>
					<startLevel>80</startLevel>
					<aggregateFeatures>true</aggregateFeatures>
					<resolver>(obr)</resolver>
					<checkDependencyChange>true</checkDependencyChange>
					<failOnDependencyChange>false</failOnDependencyChange>
					<logDependencyChanges>true</logDependencyChanges>
					<overwriteChangedDependencies>true</overwriteChangedDependencies>
					<includeTransitiveDependency>false</includeTransitiveDependency>
					<finalName>${project.artifactId}</finalName>
				</configuration>
			</plugin>
		</plugins>

````

Excluding all transitive dependencies in the dependency block seems just to be a workaround.

**Comment by lburgazzoli** [comment]
You can do it also in configuration blocks you can apply it to all the dependencies for a given configuration.

**Comment by eschizoid** [comment]
Something like the following:

````
configurations.all {
    transitive = false
}
````

**Comment by lburgazzoli** [comment]
Yep , or per configuration

```
configurations {
    jpasskit {
        transitive = false
    }
}
```

**Comment by lburgazzoli** [comment]
I've just released a v0.0.15 wich adds a little dsl for add multiple configurations to a feature overwriting the defaults:

```groovy
        feature {
            feature name = 'my-feature'
            description = 'My Feature'
            version = "1.0"

            configurations {
                add "commons_io"
                add "jpasskit"
            }
        }
```

**Comment by eschizoid** [comment]
@lburgazzoli great I think that is more than enough. Thank you.
